spharpy/samplings/samplings.py

```diff
@@ -640,18 +639,17 @@ def eigenmike_em32():
 def eigenmike_em64():
     """Microphone positions of the Eigenmike em64 by mhacoustics.
 
-    according to
-    the Eigenmuke user manual on the homepage [#]_.
-
-    References
-    ----------
-    .. [#]  Eigenmike em64 User Manual, https://eigenmike.com/sites/default/files/documentation-2024-09/getting%20started%20Guide%20to%20em64%20and%20ES3%20R01H.pdf
+    The data are according to the Eigenmike user manual on the homepage [#]_.
 
     Returns
     -------
     sampling : SamplingSphere
         SamplingSphere object containing all sampling points
 
+    References
+    ----------
+    .. [#]  Eigenmike em64 User Manual, https://eigenmike.com/sites/default/files/documentation-2024-12/em64%20Eigenbeam%20Datasheet_R01B.pdf
```

Comment:
```suggestion
    .. [#]  Eigenmike em64 Getting Started Guide, https://eigenmike.com/sites/default/files/documentation-2024-09/getting%20started%20Guide%20to%20em64%20and%20ES3%20R01H.pdf
```
Why did you change the link?